### PR TITLE
Add Style Transfer Model

### PR DESCRIPTION
**Pull request by zhanghang1989**

### CODE REVIEW

**Comment by mli** [comment]
Job PR-146-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-146/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-146-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-146/4/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-146-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-146/5/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-146-6 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-146/6/index.html 
Code coverage of this PR:  vs. Master:

File: docs/tutorials/style_transfer/demo_msg_net.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# --------------------------------
+#
+# download the example images
+content_url = 'https://github.com/dmlc/web-data/blob/master/mxnet/example/' + \
+    'style_transfer/images/content/venice-boat.jpg?raw=True'
+style_url = 'https://github.com/dmlc/web-data/blob/master/mxnet/example/' + \
+    'style_transfer/images/styles/candy.jpg?raw=True'
+content_fn = 'venice-boat.jpg'
+style_fn = 'candy.jpg'
+gluoncv.utils.download(content_url, content_fn)
```
Comment (hetong007) [inline docs/tutorials/style_transfer/demo_msg_net.py]: Please explicitly call `path_name` as in https://github.com/dmlc/gluon-cv/pull/133/files .

Or alternatively, use the url with prefix `raw.githubusercontent.com`.

**Comment by mli** [comment]
Job PR-146-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-146/7/index.html 
Code coverage of this PR:  vs. Master:

File: docs/tutorials/style_transfer/demo_msg_net.py (hunk 1 above)
Comment (zhanghang1989) [reply docs/tutorials/style_transfer/demo_msg_net.py]: Do you suggest using `path=content_fn ` instead of positional params `content_fn`?

**Comment by mli** [comment]
Job PR-146-8 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-146/8/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-146-9 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-146/9/index.html 
Code coverage of this PR:  vs. Master:

**Comment by zhreshold** [comment]
I still think segbase is a bad name which should not be exposed at all.

**Pull request closed without merging**